### PR TITLE
feat(Tabs): New design w/ indication prop

### PR DESCRIPTION
**Pull request by maraisr**

### CODE REVIEW

File: lib/components/Tabs/stories.tsx [hunk 1]
```diff
@@ -22,4 +25,22 @@ storiesOf('Components|Tabs', module)
 			<Tab title={'A'}>a</Tab>
 			<Tab title={'B'}>b</Tab>
 		</Tabs>
+	))
+	.add('with number', () => (
+		<Tabs>
+			<Tab title="Leads" indication={getRandomInt(6, 10)}>
+				Content a
+			</Tab>
+			<Tab title="Claimed" indication={getRandomInt(1, 3)}>
```
Comment (amir-zahedi) [inline lib/components/Tabs/stories.tsx]: Isn't randomising these numbers going to cause unnecessary chromatic whinging?

File: lib/components/Tabs/style.scss [hunk 2]
```diff
@@ -1,24 +1,61 @@
...
 
 .nav-item {
+	$sel: &;
+
 	display: block;
 	outline: none;
 	width: auto;
 	border: none;
-	margin-bottom: -1px;
+	margin: 0 0 -1px;
```
Comment (amir-zahedi) [inline lib/components/Tabs/style.scss]: Looks like 1px does not quite blend the two lines. I think we need -2px

File: lib/components/Tabs/style.scss [hunk 3]
```diff
@@ -1,24 +1,61 @@
...
 	background: transparent;
 	font-size: var(--global--fontSize--4);
-	padding: var(--global--space--3) var(--global--space--4);
-	color: getColour(gray, 400);
+	font-weight: 700;
+	line-height: 18px;
+	padding: calc(var(--global--space--3) + 1px) 0;
+	color: getColour(gray, 500);
 	cursor: pointer;
 	border-bottom: 2px solid transparent;
```
Comment (amir-zahedi) [inline lib/components/Tabs/style.scss]: Maybe we can use a css var to keep this border width and negative margining consistent

File: lib/components/Tabs/stories.tsx (hunk 1 above)
Comment (maraisr) [reply lib/components/Tabs/stories.tsx]: :100:

**Pull request merged**